gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManagementTaskStreamImpl.java

```diff
@@ -124,9 +131,18 @@ public DagTask next() {
       while (true) {
         try {
           DagActionStore.DagAction dagAction = this.dagActionQueue.take();
-          LeaseAttemptStatus leaseAttemptStatus = retrieveLeaseStatus(dagAction);
-          if (leaseAttemptStatus instanceof LeaseAttemptStatus.LeaseObtainedStatus) {
-            return createDagTask(dagAction, (LeaseAttemptStatus.LeaseObtainedStatus) leaseAttemptStatus);
+          // create triggers for original (non-reminder) dag actions of type ENFORCE_JOB_START_DEADLINE and ENFORCE_FLOW_FINISH_DEADLINE
+          // reminder triggers are used to inform hosts once the deadline for ENFORCE_JOB_START and ENFORCE_FLOW_FINISH passed
```

Comment:
missing "_DEADLINE" also nit: let's use a multi-line comment /*....*/

Reply:
yea, either "deadline for ENFORCE_JOB_START" or ENFORCE_JOB_START_DEADLINE